source/_components/rainbird.markdown

```diff
@@ -0,0 +1,43 @@
+---
+layout: page
+title: "Rainbird"
+description: "Instructions on how to integrate your Rainbird WiFi LNK within Home Assistant."
+date: 2017-08-25 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: rainbird.png
+ha_category: Hub
+ha_release: 0.50
+ha_iot_class: "Local Polling"
+---
+
+This `Rainbird` components alows interacting with [WiFi LNK](http://www.rainbird.com/landscape/products/controllers/LNK-WiFi.htm) module of the Rainbird Irrigation system in Home Assistant.

```

Comment:
**Spelling & Grammar**:
- `This Rainbird` -> `This rainbird` (component name is lower case)
- `components` -> `component`
- `alows` -> `allows`
- `WiFi LNK` -> `LNK WiFi` (according to the manufacturer)
- `of the Rainbird` -> `of the Rain Bird`